03-transactions.md

```diff
@@ -109,7 +111,45 @@ If a revoked commitment transaction is published, the other party can spend this
 
 #### `to_remote` Output
 
-This output sends funds to the other peer and thus is a simple P2WPKH to `remotepubkey`.
+This output sends funds to the other peer, thus is not encumbered by a
+revocation private key.
+
+If `option_simplified_commitment` applies to the commitment transaction, the `to_remote` output is delayed similarly to the `to_local` output, and is to a fixed key:
+
+    `to_self_delay`
+    OP_CSV
+    OP_DROP
+    <remote_pubkey>
+    OP_CHECKSIG
+
+The output is spent by a transaction with `nSequence` field set to `to_self_delay` (which can only be valid after that duration has passed) and witness:
+
+    <remote_sig>
+
+Otherwise, this output is a simple P2WPKH to `remotepubkey`.
+
+
+#### `to_local_pushme` and `to_remote_pushme` Output (option_simplified_commitment)
+
+This output can be spent by the local and remote nodes respectively to provide incentive to mine the transaction, using child-pays-for-parent.  They are only added if the `to_local` and `to_remote` outputs exist, respectively.
```

Comment:
"They are only added if the `to_local` and `to_remote` outputs exist, respectively." Hmmmm...I think we need to move to a world where dust limit must be <= channel_reserve + fee on commitment tx, then. If you imagine adding a number of HTLCs to increase the absolute fee and then add a large HTLC it could drop the to_local/to_remote output, but you'll still need to be able to get the tx confirmed using CPFP to broadcast it on HTLC timeout/claim.

Reply:
So, 'funder must keep >= dust-limit + worst-case commitment tx fees'?  (as well as the standard reserve requirements)?   This would mean that even if fundee adds 483 HTLCs at once (a case which currently can potentially dip into funders' reserves to pay fees) the funder's output would not be dust.

This is probably OK with the minimal feerate proposed here (21323 satoshi), but @Roasbeef wanted to separate "fixed minimal fee" from "simplified commitment" option so we weren't so reliant on propagation changes to make this work.  But I guess in that case we could ignore this corner case anyway...